src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -308,9 +309,13 @@ private static String getChecksumFileName(Path base, String customPart, String o
 
     private void downloadJavaZip(String target, Path substratePath) throws IOException {
         Logger.logDebug("Process zip javaStaticSdk, target = "+target);
-        String javaZip = JAVA_STATIC_ZIP
-                .replace("${version}", configuration.getJavaStaticSdkVersion())
-                .replace("${target}", target);
+
+        String javaZip = Strings.substitute( JAVA_STATIC_ZIP, Map.of(
+            "version",  configuration.getJavaStaticSdkVersion(),
+            "target",  target
+        ));
+
```

Comment:
remove extra blank lines

Reply:
I think those provide more clarity on steps in the code though

Reply:
Sure, but one line should be enough?